cl/opinion_page/views.py

```diff
@@ -176,7 +239,14 @@ async def court_publish_page(request: HttpRequest, pk: int) -> HttpResponse:
                 request, "Error submitting form, please review below."
             )
     return TemplateResponse(
-        request, "publish.html", {"form": form, "private": True, "pk": pk}
+        request,
+        "publish.html",
+        {
+            "court_image": f"img/{pk}.jpg",
```

Comment:
maybe we cant use seal rookery because of CORS - but wouldnt that be nicer than storing all these images

Reply:
If CORS is the problem, we can add seals.free.law to our allowed list in our settings and fix that.

Reply:
great.  thanks.  Currently its a problem as we actually use seals on the Maine uploader page but its blocked from loading

Reply:
The problem was with the CSP, you just need to add the domain seal rockery to CSP_IMG_SRC in security.py